backend/open_webui/retrieval/loaders/youtube.py

```
@@ -110,11 +110,37 @@ def load(self) -> List[Document]:
         for lang in self.language:
             try:
                 transcript = transcript_list.find_transcript([lang])
+                if transcript.is_generated:
+                    log.debug(f"Found generated transcript for language '{lang}'")
```

Ruff (0.11.9)

Logging statement uses f-string

(G004)